paths.go

```diff
@@ -27,12 +27,24 @@ type Paths struct {
 	electronUnzipSrc       string
 	provisionStatus        string
 	vendorDirectory        string
+	VersionAstilectron     string
+	VersionElectron        string
 }
 
 // newPaths creates new paths
 func newPaths(os, arch string, o Options) (p *Paths, err error) {
+	if o.VersionAstilectron == "" {
```

Comment:
Unless I missed something, this should be removed. Indeed, defaults are already set in `Astilectron`'s `New()` method.

Reply:
removed - I had added due to passing empty Options{} from tests. Making sure to populate Options{} in the tests with default versions.